src/purify.js

```diff
@@ -130,11 +128,8 @@ function createDOMPurify(window = getGlobal()) {
     }
   }
 
-  let trustedTypesPolicy = _createTrustedTypesPolicy(
-    trustedTypes,
-    originalDocument
-  );
-  let emptyHTML = trustedTypesPolicy ? trustedTypesPolicy.createHTML('') : '';
+  let trustedTypesPolicy;
+  let emptyHTML = '';
```

Comment:
nit: I would revert to previous declaration 
```
const emptyHTML = trustedTypesPolicy ? trustedTypesPolicy.createHTML('') : '';
```
and move it just after either default or custom trustedTypesPolicy is initialized.

Reply:
I believe we would be missing some functionality if we make it a const and initialized one time. Particularly this kind of usage:

```
DOMPurify.sanitize(dirty1);
DOMPurify.sanitize(dirty2, {TRUSTED_TYPES_POLICY: policy});
```

Since the first initialization is without the cfg, which may or may not succeed to produce a policy and at which point `emptyHTML` may end up not being signed, the 2nd call may not work as expected because some methods do rely on `emptyHTML` when they build the DOM tree.

There is, however, an alternative to initializing this. Looking at the existing `trustedTypes` API in Chrome, there seems to be a `trustedTypes.emptyHTML` signed value built into the API so this declaration would not be needed anymore. We could simply declare it as

```
const emptyHTML = trustedTypes? trustedTypes.emptyHTML : ''
```

I did not want to make that change because I do not have enough information to know if the API is considered stable at the moment and if this will be supported. Should be an easy improvement for a follow up PR in case it is.

Reply:
Let's keep current change then if it's intentional.